bin/build_javascript.js

```diff
@@ -21,6 +21,8 @@ const files = [
     // custom handling for StimulusBundle
     'src/StimulusBundle/assets/src/loader.ts',
     'src/StimulusBundle/assets/src/controllers.ts',
+    // custom handling for Bridge
+    ...glob.sync('src/*/src/Bridge/*/assets/src/*controller.ts'),
     ...glob.sync('src/*/assets/src/*controller.ts'),
```

Comment:
```suggestion
    // custom handling for Map bridges
    ...glob.sync('src/Map/src/Bridge/*/assets/src/*controller.ts'),
    ...glob.sync('src/Map/assets/src/*controller.ts'),
```

Reply:
Wildcarding every bridges for every components is not a good thing?

Reply:
We don't have other bridges, and this feels a bit "large" to me 

```
    ...glob.sync('src/*/assets/src/*controller.ts'),
```

But nothing really decisive or blocking here :)

Reply:
Not sure to understand your last comment :/ 

`...glob.sync('src/*/assets/src/*controller.ts')` is already present, but it won't match controllers from Bridges, `src/*/assets` expand to `src/{Autocomplete,Cropperjs,Map,...}/assets`, so we need `...glob.sync('src/*/src/Bridge/*/assets/src/*controller.ts'` (or `...glob.sync('src/Map/src/Bridge/*/assets/src/*controller.ts')` if we really want to be strict)